src/state/move-to-next-index/move-to-next-index-types.js

```diff
@@ -11,7 +11,7 @@ import type {
 export type Args = {|
   isMovingForward: boolean,
   draggableId: DraggableId,
-  previousPageCenter: Position,
+  previousPageBorderBoxCenter: Position,
```

Comment:
Can we be importing Position from `css-box-model` here (and anywhere else Position is imported)?